source/index.html.md

```diff
@@ -4144,7 +4145,7 @@ curl --request POST \
   --header 'accept: application/json' \
   --header 'authorization: Bearer {access-token}' \
   --header 'content-type: application/json' \
-  --data '{"title":"Subscription Plan A","terms":{"per_payout":{"min_amount":null,"max_amount":10000},"per_frequency":{"days":7,"max_amount":1000000}},"metadata":{"custom_key":"Custom string","another_custom_key":"Maybe a URL"}}'
+  --data '{"title":"Subscription Plan A","terms":{"per_payout":{"min_amount":0,"max_amount":10000},"per_frequency":{"days":7,"max_amount":1000000}},"metadata":{"custom_key":"Custom string","another_custom_key":"Maybe a URL"}}'
```

Comment:
I wonder if the original intent was to demonstrate that `min_amount` isn't required, in which case it could be omitted. 🤷 

Regardless, `0` amount doesn't really make sense. 

(ditto below)

Reply:
Good catch. I changed the schema type to allow integer or null, and then forgot to update this example. Will fix 👍